luna/pretrained_models/alexnet.py

```diff
@@ -2,13 +2,18 @@
 A Keras Implementation for Alexnet from
 https://github.com/heuritech/convnets-keras/blob/master/convnetskeras/convnets.py
 """
+from  pathlib import Path
 from keras import backend as K
 from keras.engine import Layer
 from keras.models import Model
 from keras.layers import (Activation, Dense, Dropout, Flatten, Input,
                           concatenate, Lambda)
-from keras.layers.convolutional import (Convolution2D, MaxPooling2D,
+from keras.layers.convolutional import (MaxPooling2D,
                                         ZeroPadding2D)
+from keras.utils.data_utils import get_file
+from tensorflow.keras.layers import Conv2D
```

Comment:
Could merge with the other layers import.

Reply:
Still valid.

Reply:
tensorflow.keras.layers can be merged. I cannot find a merge for get_file
p.s. Alexnet does not work (properly) yet